adapter/slack.py

```diff
@@ -3,12 +3,14 @@
 from slack_bolt import App
 from slack_bolt.adapter.fastapi import SlackRequestHandler
 from slack_sdk.errors import SlackApiError
+from chat.chat import ChatOpenAI
 
 class SlackAdapter:
-    def __init__(self, app: App) -> None:
+    def __init__(self, app: App, chatbot: ChatOpenAI) -> None:

```

Comment:
So for now, the integration is only with OpenAI, right? If we want to use another chatbot model, do we need to add a new chatbot parameter in the Slack adapter constructor?

Reply:
Great catch, it shouldn't be `ChatOpenAI` but rather a `Chat` class. Since it could be ChatOpenAI or ChatAnthropic, etc. Will fix this soon!